tools/go.mod

```diff
@@ -4,12 +4,12 @@ go 1.15
 
 require (
 	github.com/BurntSushi/toml v0.3.1
-	github.com/buildpacks/imgutil v0.0.0-20201211223552-8581300fe2b2
+	github.com/buildpacks/imgutil v0.0.0-20210119193758-3436c122b313
```

Comment:
I don't quite understand how this `tools/` became pinned to this version. I would have expected we may need a `replace golang.org/x/sys => golang.org/x/sys v0.0.0-20200523222454-059865788121` in this `go.mod` as well. 😕

Reply:
I believe since imgutil is already pinned, everything should work as expected: https://github.com/buildpacks/imgutil/commit/3436c122b31395e516adc4cc566823b2eb6d13fc

`tools` is used by our Makefile, so it everything is compiling with this version of imgutil.